roles/neutron-common/templates/etc/neutron/plugins/ml2/ml2_plugin.ini

```diff
@@ -19,6 +19,9 @@ tenant_network_types = {{ neutron.tenant_network_type }}
 {% endif %}
 mechanism_drivers = {{ mechanism_drivers|join(',') }}
 
+{% set extension_drivers = ['port_security'] %}
+extension_drivers = {{ extension_drivers|join(',') }}
```

Comment:
@j2sol This is inline with mechanisms driver. If we had to enable more neutron extensions later, then it would be a simple addition. We can define that extension to be true or False in defaults.yml and add it to the list in ml2_plugin.ini template.